### ISSUE TITLE
Could you add the license field to package.json

### ISSUE THREAD

**Issue by hnakamur** [evanw/zeroscript#11]
First of all, thanks for a great software!

Could you add the license field to package.json?
[license field in package.json | npm Documentation](https://docs.npmjs.com/files/package.json#license)

The following page may help for choosing a license.
[Choose an open source license - Choose a License](http://choosealicense.com/)

I hope you choose a permissive license like BSD, MIT, or Apache.
Thanks!

**Comment by ghost** [comment]
I agree. It says it's open source in the readme but lacks a license to specify the type.

**Comment by peirick** [comment]
Please add a License :)
